workos/session.py

```diff
@@ -0,0 +1,174 @@
+import json
+from typing import Any, Dict, List, Optional, Union
+import jwt
+from jwt import PyJWKClient
+from cryptography.fernet import Fernet
+
+from workos.types.user_management.session import (
+    AuthenticateWithSessionCookieFailureReason,
+    AuthenticateWithSessionCookieSuccessResponse,
+    AuthenticateWithSessionCookieErrorResponse,
+    RefreshWithSessionCookieErrorResponse,
+    RefreshWithSessionCookieSuccessResponse,
+)
+
+
+class SessionModule:
```

Comment:
The only reason we have other classes called "*Module" is because they're protocol (abstract) classes. We can just call this `Session`.